### DISCUSSION TITLE
Legal considerations for distributing binary builds

### DISCUSSION THREAD

**Discussion by RichardFrangenberg** [AcademySoftwareFoundation/OpenRV#95] [General]
Is there any update on the legal status of distributing binary builds of Open RV?
It was mentioned here that there will be updates coming regarding this topic: [guillaume-brossard ](https://github.com/AcademySoftwareFoundation/OpenRV/discussions/67?sort=top#discussioncomment-5014917)

Specifically we're interested in the case of sharing the Open RV builds with clients, who are working with us. We are developing plugins for Open RV and would like to share Open RV and the plugins together with clients.

Which 3rd party licenses are causing the most uncertainty?

**Comment by compaga** [comment]
Hi RichardFrangenberg,

Open RV TSC isn’t ready to trigger this discussion with ASWF legal advisor because there are multiple steps that we wanted to complete before. CI/CD for instance. This is also a topic that we might want to discuss at Open Review Initiative level.
That being said, if you would like to redistribute it for your company needs, you can work this out with your own legal advisor. 

Alain

**Reply by RichardFrangenberg** [reply]
Thanks for clarifying, Alain.
